### PR TITLE
Add pony.town presence

### PR DESCRIPTION
**Pull request by Tschrock**
Adds presence for the pixel game https://pony.town/

1598650538-RN95
1598650519-MkWo

### CODE REVIEW

File: websites/P/PonyTown/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "id": "163300806769115136"
+  },
+  "service": "PonyTown",
+  "description": {
+    "en": "A game of ponies building a town."
+  },
+  "url": ["pony.town", "event.pony.town"],
+  "version": "1.0.0",
+  "logo": "https://p.cp3.es/pony.town-icon-512.png",
+  "thumbnail": "https://p.cp3.es/pony.town-thumbnail2.png",
```
Comment (Bas950) [inline websites/P/PonyTown/dist/metadata.json:14]: upload to imgur

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by Bas950** [comment]
also update your fork

**Comment by Tschrock** [comment]
rebased & imgur'd

File: websites/P/PonyTown/presence.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
+var presence = new Presence({
+  clientId: "748780200006778892"
+});
```
Comment (Bas950) [inline websites/P/PonyTown/presence.ts:3]: Somehow you are not getting any deepscan issues
but no-var

File: websites/P/PonyTown/presence.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+enum PlayingState {
+  UNKNOWN,
+  CHARACTER_CREATOR,
+  IN_GAME
+}
+
+let lastSeenCharacterName = "New Character";
+let lastSeenServerName = "Unknown Server";
+let lastSeenState = PlayingState.UNKNOWN;
+let lastStateTime = Date.now();
```
Comment (Bas950) [inline websites/P/PonyTown/presence.ts:23]: one-var (merge lets to just 1)

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/P/PonyTown/presence.ts [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+});
+
+const selectors = {
+  CHARACTER_CREATOR: 'pony-town-app main > character',
+  GAME_ELEMENT: '#app-game',
+  SERVER_NAME: 'play-box .btn-success > div > span',
+  CHARACTER_NAME: '.character-select > input',
+  IN_GAME_SERVER_NAME: '#app-game span[title="Current server"]',
+  IN_GAME_CHARACTER_NAME: '.character-select-list > .selected .character-name'
+}
```
Comment (doomlerd) [inline websites/P/PonyTown/presence.ts:12]: Combine those two consts and fix any issues that _may_ be reported by DeepScan.

**Review by doomlerd — CHANGES_REQUESTED** [review]

File: websites/P/PonyTown/presence.ts [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  characterName: "New Character",
+  serverName: "Unknown Server",
+  time: Date.now()
+}
+
+// On presence update request
+presence.on("UpdateData", async () => {
+
+  // Check if the user is in-game
+  const gameElement = document.querySelector(selectors.GAME_ELEMENT)
```
Comment (SeMiD) [inline websites/P/PonyTown/presence.ts:31]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113511)

File: websites/P/PonyTown/presence.ts (hunk 4 above)
Comment (SeMiD) [inline websites/P/PonyTown/presence.ts:12]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113511)

File: websites/P/PonyTown/presence.ts [hunk 6]
```diff
@@ -0,0 +1,116 @@
+const presence = new Presence({
+  clientId: "748780200006778892"
+});
+
+const selectors = {
```
Comment (SeMiD) [inline websites/P/PonyTown/presence.ts:5]: Codacy found an issue: [Combine this with the previous 'const' statement.](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113511)

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy issues

**Comment by doomlerd** [comment]
Closing due to inactivity.

**Pull request closed without merging**